src/Sql/Sql.Test/ScenarioTests/SqlTestsBase.cs

```diff
@@ -69,8 +69,8 @@ protected void RunPowerShellTest(params string[] scripts)
                 {"Microsoft.Features", null},
                 {"Microsoft.Authorization", null},
                 {"Microsoft.Network", null},
-                {"Microsoft.KeyVault", null}
-
+                {"Microsoft.KeyVault", null},
+                {"Microsoft.Sql", null }
```

Comment:
You cannot add a pernmissive matcher for Sql tests - this essentially means that any api-version will match for the Sql provider.  This is not allowed for the RP that is the target of the tests